app/routes/methoden.ablaeufe-aufgaben-erfassen.tsx

```diff
@@ -0,0 +1,138 @@
+import { useLoaderData } from "react-router";
+import { twJoin } from "tailwind-merge";
+import Background from "~/components/Background";
+import Card from "~/components/Card";
+import Container from "~/components/Container";
+import Header from "~/components/Header";
+import HighlightBox from "~/components/HighlightBox";
+import InfoBox from "~/components/InfoBox";
+import InterviewBanner from "~/components/InterviewBanner";
+import Tabs, { TabItem } from "~/components/Tabs";
+import VisualisationItem from "~/components/VisualisationItem";
+import {
+  type ContentItem,
+  methodsTasksProcesses,
+} from "~/resources/content/methode-ablaeufe-aufgaben-erfassen";
+import {
+  fetchStrapiData,
+  visualisationFields,
+  Visualisierung,
+} from "~/utils/strapiData.server";
+
+const GET_VISUALISATIONS_QUERY = `
+${visualisationFields}
+query GetVisualisierungen($filters: VisualisierungFiltersInput) {
+  visualisierungen(filters: $filters) {
+    ...VisualisationFields
+  }
+}
+`;
+
+export const loader = async () => {
+  const visualisationsData = await fetchStrapiData<{
+    visualisierungen: Visualisierung[];
+  }>(GET_VISUALISATIONS_QUERY, {
+    filters: {
+      documentId: {
+        in: methodsTasksProcesses.tabs
+          .flatMap(({ content }) =>
+            content.filter((item) => item.element === "visualisation"),
+          )
+          .map(({ documentId }) => documentId),
+      },
+    },
+  });
+
+  if ("error" in visualisationsData) {
+    // eslint-disable-next-line @typescript-eslint/only-throw-error
+    throw new Response(visualisationsData.error, { status: 400 });
```

Comment:
I would argue that this should have status 500 since it is not the users fault but an internal server error.

Reply:
Additionally, I would say we should use another error message here and not forward the strapi error since it is shown to the user and might contain information we do not want to disclose. So the approach I would suggest: log the strapi error response and throw an error like "Hier hat leider etwas nicht funktioniert, das tut uns Leid."